Makefile

```diff
@@ -297,3 +298,8 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+get-iso:
+ifeq (,$(wildcard ./testdata/Fedora.iso))
+	curl -L https://download.fedoraproject.org/pub/fedora/linux/releases/36/Server/x86_64/iso/Fedora-Server-netinst-x86_64-36-1.5.iso -o testdata/Fedora.iso
```

Comment:
can you also add a test for RHEL iso?

Reply:
no, image it's not public

Reply:
@eloycoto @bardielle Can we resolve this?